src/App.jsx

```diff
@@ -2,17 +2,20 @@ import { useState } from 'react';
 import './App.scss';
 
 export const App = () => {
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+
+  const currentCount = (numAdd) => {
```

Comment:
Try to name functions in a more descriptive way. You should understand what the function does by only reading its name.

`currentCount` is a bit unclear though. It would be better to use something like `increaseCountByValue`. Just keep it in mind for the future)